addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -3274,7 +3274,18 @@ const SnippetOptionWidget = Widget.extend({
 
         function applyCSS(cssProp, cssValue, styles) {
             if (!weUtils.areCssValuesEqual(styles[cssProp], cssValue, cssProp, this.$target[0])) {
-                this.$target[0].style.setProperty(cssProp, cssValue, 'important');
+                this.$target[0].style.setProperty(cssProp, cssValue);
+                // If change had no effect then make it important.
+                if (params.extraClass) {
+                    this.$target[0].classList.add(params.extraClass);
+                }
```

Comment:
If we were sure `extraClass` will remain limited to boostrap classes with `!important`, we could just enable `!important` whenever there is an `extraClass` instead of recomputing the styles with the added class.